src/values.js

```diff
@@ -105,6 +107,7 @@ const highlight = blue
 const success = green
 const warning = orange
 const danger = red
+const focused = '#2EB6EA'
```

Comment:
Ideally rug should be component/app agnostic and more an implementation of the core design system values. I think `focused` might start becoming a little too specific.